pulser/tests/test_sequence.py

```diff
@@ -327,8 +327,9 @@ def test_sequence():
     seq.phase_shift(np.pi, "q0", basis="ground-rydberg")
 
     with patch("matplotlib.pyplot.show"):
-        with patch("matplotlib.pyplot.savefig"):
-            seq.draw(fig_name="my_sequence.pdf")
+        with patch("matplotlib.figure.Figure.savefig"):
+            seq.draw(fig_name_seq="my_sequence.pdf")
+            seq.draw(draw_register=True, fig_name_reg="my_register.pdf")
```

Comment:
Did you check this locally? Is it not saving these files?

Reply:
Yes it's not saving them, that's the reason why I changed it from `patch("matplotlib.pyplot.savefig")` to `patch("matplotlib.figure.Figure.savefig")`

Reply:
Sure, I just wanted to make sure because `patch` is a bit finicky to work with sometimes